tests/integration/test_resourcegroups.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+import json
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import short_uid
+
+
+class TestRESROUCEGROUPIntegrations(unittest.TestCase):
+    def setUp(self):
+        self.resource_group_client = aws_stack.connect_to_service('resource-groups')
+
+    def cleanUp(self, name):
+        self.resource_group_client.delete_group(GroupName=name)
+
+    def test_create_group(self):
+        name = 'resource_group-{}'.format(short_uid())
+        response = self.resource_group_client.create_group(
```

Comment:
nit: Looks like lines 16-31 are duplicated 4 times in this class - can we either put this common logic into a util method, or simply combine the 4 test methods into a single test method? (I think testing create/get/list/delete in a single test method would make sense in this case..)

Reply:
Done